packages/postgres-database/tests/test_utils_projects.py

```diff
@@ -51,6 +53,26 @@ async def registered_project(
     await _delete_project(connection, project["uuid"])
 
 
+@pytest.mark.parametrize("expected", (datetime.now(tz=timezone.utc), None))
+async def test_get_project_trashed_at_column_can_be_converted_to_datetime(
+    asyncpg_engine: AsyncEngine, registered_project: dict, expected: datetime | None
+):
+    project_id = registered_project["uuid"]
+
+    async with transaction_context(asyncpg_engine) as conn:
```

Comment:
transation_context? is this some new thing?, so this wraps begin. ok

Reply:
Ah this is the "new" thing i introduced when you left for holidays and wanted to talk you about. See https://github.com/itisfoundation/osparc-simcore/blob/52771a9e88ff5ef9962a214342bca22c4c1c27f2/packages/postgres-database/tests/test_utils_repos.py